README.md

```diff
@@ -1,3 +1,10 @@
+### ** New Config Connector is coming in 2024 Q4 **
+
+We are developing a faster, more Kubernetes native and more reliable Config Connector reconciliation approach(Direct Controller).
+Thus, we decide to hold off any new PRs about resource coverage until mid Q4 2024. 
```

Comment:
For resources which are Beta *and* are currently of type TF I think we should probably continue supporting new PRs.

Reply:
Updated.